pkg/core/pmongo.go

```diff
@@ -231,3 +231,18 @@ func (s *DBConnection) UpdateFieldValue(ctx context.Context, query Q, collection
 
 	return err
 }
+
+// Find the latest data based on given query
+func (s *DBConnection) FindLatest(ctx context.Context, query Q, document Document) error {
+	opts := &options.FindOneOptions{
+		Sort: map[string]int{"createdDate": -1},
```

Comment:
Can't we use `_id` instead?

Reply:
I think Sort is done based on created date to find latest auth log or subsequent document entry.

Reply:
@chawocecil  @samirkhanal52.  Please refer to [this](https://www.mongodb.com/docs/manual/reference/method/ObjectId/#ObjectIDs-DocumentTimestamps).

> 12-byte ObjectId consists of:
> - A 4-byte timestamp, representing the ObjectId's creation, measured in seconds since the Unix epoch.